lib/ship.rb

```diff
@@ -0,0 +1,17 @@
+class Ship
+    attr_reader :name, :length, :health
+    def initialize(name, length)
+        @name = name
+        @length = length
+        @health = length
```

Comment:
I think this makes the most sense right now bc health is dependent on the length of the ship!